WebDevStudios/wd_s
# Feature/#522 css source maps

Closes #522

### DESCRIPTION ###
Updates our npm scripts to bring source maps back into play.

I was able to do this with just one pretty small change:
```"scss": "node-sass src/scss/style.scss -o dist/css --source-map true"```

This seems to work running `npm i`, `npm run build`, and `npm run watch`. I tried running `npm i` with the theme installed in the same directories I was experiencing errors previously in #527 and #528.

I'd love to have @jeseph test this branch as well since the initial reporting of this issue came from them on Twitter. I know it's working for me now where it wasn't before, but we all kind of experienced the install and build error in different ways so I want to try and make sure all bases are covered.

We'll test and review internally as well.

### SCREENSHOT ###
![view source of an element](https://d.pr/i/VXo3g6+)
![click-through of the view source link](https://d.pr/i/7b8Ywj+)

### OTHER ###

### STEPS TO VERIFY ###
1. Checkout the branch
2. Run `npm i` in a fresh install
  a. View the site in a browser and inspect elements to see where the styles are living
3. Run `npm run build`
  a. View the site in a browser and inspect elements to see where the styles are living
4. Run `npm run watch` and make changes to files + save
  a. View the site in a browser and inspect elements to see where the styles are living

### DOCUMENTATION ###
Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

Don't think so – this is all under the hood and everyone should (hopefully!) just be able to enjoy it.
